tests/pytest/test_logic_array.py

```diff
@@ -13,6 +13,29 @@ def test_logic_array_constructor():
     with pytest.raises(ValueError):
         LogicArray([object()])
 
+    assert LogicArray(0) == LogicArray("0")
+    assert LogicArray(0xA7) == LogicArray("10100111")
+    assert LogicArray(-1) == LogicArray("1")
+
+    assert LogicArray(10, Range(5, "downto", 0)) == LogicArray("001010")
+    assert LogicArray(-2, Range(5, "downto", 0)) == LogicArray("111110")
+    with pytest.raises(ValueError):
+        LogicArray(10, Range(1, "to", 3))
+
+
+def test_logic_array_properties():
+    assert LogicArray(0).integer == 0
+    assert LogicArray(0).signed_integer == 0
+    assert LogicArray(0).binstr == "0"
+    assert LogicArray(10).integer == 10
+    assert LogicArray(10).signed_integer == -6
+    assert LogicArray(10).binstr == "1010"
+    assert LogicArray(-6).integer == 10
+    assert LogicArray(-6).signed_integer == -6
+    assert LogicArray(-6).binstr == "1010"
+    assert LogicArray(0).is_resolvable
+    assert not LogicArray("XX").is_resolvable
```

Comment:
```suggestion
    assert not LogicArray("1X1").is_resolvable
```
to make sure that not just the first or last element is taken for the resolvability check?